horace_core/sqw/coord_transform/@aProjection/aProjection.m

```diff
@@ -67,11 +69,15 @@
         % true, though testing or the projection used to identify position
         % of q-dE point in q-dE space may set this property to false.
         do_3D_transformation;
+        % direct access to different parts of 4-component label celarray.
+        % sets up appropriate element of such array. Do not have a getter.
+        % Do retrievel label as a whole.
+        lab1;
+        lab2;
+        lab3;
+        lab4;
```

Comment:
At least write "label" but better to say what they are.

Reply:
it is explained above the row and is here for compatibility with previous Toby's code